challenges/src/main/java/linkedlist/LinkedList.java

```diff
@@ -0,0 +1,82 @@
+
+package linkedlist;
+
+public class LinkedList {
+
+    // makes Node head stuff 'global' within public class
+    // Within your LinkedList class, include a head property. Upon instantiation, an empty Linked List should be created.
+    Node head;
+
+    // Create a Node class that has properties for the value stored in the Node, and a pointer to the next Node.
+    class Node {
+
+        int valueData;
+        Node next;
+
+
+        Node(int valueData, Node next) {
+
+            this.valueData = valueData;
+            this.next = next;
+
+        }
+
+    }
+
+    // Define a method called insert which takes any value as an argument and adds a new node with that value to the head of the list with an O(1) Time performance.
+    public void insertHead(int headValue){
+
+        // New Head instantiation
+        Node HeadNode = new Node(headValue, null);
+
+        // Retain old list data
+        Node OldHead = this.head;
+
+        // Create new head
+        this.head = HeadNode;
+
+        // Assign new head's "next" to old list data
+        this.head.next = OldHead;
+
```

Comment:
this can shortened to a one liner;
this.head = new Node(headValue, this.head);
But is totally optional. If that's confusing, then just ignore it.

Reply:
Not at all, thanks for pointing out a way I can use to make my code more presentable.